boards/stm32mp157c-dk2/Kconfig

```diff
@@ -15,3 +15,10 @@ config BOARD_STM32MP157C_DK2
     # Put defined MCU peripherals here (in alphabetical order)
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
+    select HAS_PERIPH_GPIO
```

Comment:
Shouldn't this be already selected at CPU level ?

Reply:
Yup, I think I was just trying to copy what was in the makefile `FEATURES_PROVIDED`.  Fixed.